pkgs/by-name/cy/cyrus-imapd/package.nix

```diff
@@ -0,0 +1,208 @@
+{
+  autoreconfHook,
+  bison,
+  brotli,
+  coreutils,
+  cunit,
+  cyrus-imapd,
+  cyrus_sasl,
+  enableAutoCreate ? true,
+  enableBackup ? true,
+  enableCalalarmd ? true,
+  enableHttp ? true,
+  enableIdled ? true,
+  enableJMAP ? true,
+  enableMurder ? true,
+  enableNNTP ? false,
+  enableReplication ? true,
+  enableSrs ? true,
+  enableUnitTests ? true,
+  enableXapian ? true,
+  fetchFromGitHub,
+  fetchpatch,
+  fetchurl,
+  fig2dev,
+  flex,
+  icu,
+  jansson,
+  lib,
+  libbsd,
+  libcap,
+  libchardet,
+  libical,
+  libmysqlclient,
+  libsrs2,
+  libuuid,
+  libxml2,
+  makeWrapper,
+  nghttp2,
+  openssl,
+  pcre2,
+  perl,
+  pkg-config,
+  postgresql,
+  rsync,
+  shapelib,
+  sqlite,
+  stdenv,
+  testers,
+  unixtools,
+  valgrind,
+  withLibcap ? true,
+  withMySQL ? false,
+  withOpenssl ? true,
+  withPgSQL ? false,
+  withSQLite ? true,
+  withZlib ? true,
+  wslay,
+  xapian,
+  zlib,
```

Comment:
This is really hard to make sense of. Instead of using an alphabetical order, consider using the order in which these variables appear later in the derivation, except for the feature flags, which should be placed at the bottom of the list by convention.